docs/uml-class-diagrams/database/test/DatabasePersistentStorageImplementationTests/DatabasePersistentStorageImplementationTests.puml

```diff
@@ -2,7 +2,14 @@
 
 package edu.oakland.test.database {
     class DatabasePersistentStorageImplementationTests {
-
+        + void rfidDataInIsRfidDataOut()
+        + void offsetMatchDataPoint()
+        + void locationDataPointInIsDataPointOut()
+        + void offsetMatchTrackData()
+        + void trackDataInIsTrackDataOut()
+        + void readModeInIsModeOut()
+        + void writeModeInIsModeOut()
+        + void currentSatMatchNextSat()
```

Comment:
Just a few more test cases regarding edge cases. Odds are we shouldn't hit the first two, but I think an out-of-bounds exception could be likely if we don't make a deliberate effort to handle it.
```suggestion
        + void currentSatMatchNextSat()
        + void cannotStoreNullLocationDataPoint()
        + void cannotStoreNullTrackData()
        + void locationDataPointOffsetOutOfBoundsReturnsNull()
        + void trackDataOffsetOutOfBoundsReturnsNull()
```